index.json

```diff
@@ -13380,12 +13409,14 @@
   },
   {
     "name": "pull_request",
+    "description": "Triggered when a [pull request](https://developer.github.com/v3/pulls) is `assigned`, `unassigned`, `labeled`, `unlabeled`, `opened`, `edited`, `closed`, `reopened`, `synchronize`, `ready_for_review`, `locked`, `unlocked` or when a pull request review is requested or removed.",
     "actions": [
       "assigned",
       "closed",
       "edited",
       "labeled",
       "locked",
+      "merged",
```

Comment:
@gr2m this one got added to FYI

Reply:
ah thanks for the pointer, `merged` is not a valid action